<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13908" started-at="2019-02-15T19:38:57Z" finished-at="2019-02-15T19:39:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13908" started-at="2019-02-15T19:38:57Z" finished-at="2019-02-15T19:39:11Z">
      <class name="com.learnAutomation.testcases.LoginTestCRM">
        <test-method status="PASS" signature="setupSuite()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="setupSuite" is-config="true" duration-ms="699" started-at="2019-02-16T01:08:56Z" finished-at="2019-02-16T01:08:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="setup" is-config="true" duration-ms="10063" started-at="2019-02-16T01:08:57Z" finished-at="2019-02-16T01:09:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="Testcase1()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="Testcase1" duration-ms="67" started-at="2019-02-16T01:09:07Z" finished-at="2019-02-16T01:09:07Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.sendKeys(Unknown Source)
	at com.learnAutomation.webpages.LoginPage.loginToCRM(LoginPage.java:28)
	at com.learnAutomation.testcases.LoginTestCRM.Testcase1(LoginTestCRM.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Testcase1 -->
        <test-method status="PASS" signature="screenShotOnFailure(org.testng.ITestResult)[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="screenShotOnFailure" is-config="true" duration-ms="1164" started-at="2019-02-16T01:09:07Z" finished-at="2019-02-16T01:09:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Testcase1 status=FAILURE method=LoginTestCRM.Testcase1()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenShotOnFailure -->
        <test-method status="PASS" signature="Testcase2()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="Testcase2" duration-ms="3" started-at="2019-02-16T01:09:08Z" finished-at="2019-02-16T01:09:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Testcase2 -->
        <test-method status="PASS" signature="screenShotOnFailure(org.testng.ITestResult)[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="screenShotOnFailure" is-config="true" duration-ms="1876" started-at="2019-02-16T01:09:08Z" finished-at="2019-02-16T01:09:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Testcase2 status=SUCCESS method=LoginTestCRM.Testcase2()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenShotOnFailure -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.learnAutomation.testcases.LoginTestCRM@1b604f19]" name="tearDown" is-config="true" duration-ms="715" started-at="2019-02-16T01:09:10Z" finished-at="2019-02-16T01:09:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.learnAutomation.testcases.LoginTestCRM -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
